frontend/pages/hosts/details/cards/Policies/HostPolicies.tsx

```diff
@@ -30,15 +30,32 @@ const Policies = ({
   togglePolicyDetailsModal,
   hostPlatform,
 }: IPoliciesProps): JSX.Element => {
-  if (policies.length === 0) {
-    return (
-      <Card
-        borderRadiusSize="xxlarge"
-        includeShadow
-        largePadding
-        className={baseClass}
-      >
-        <p className="card__header">Policies</p>
+  const tableHeaders = generatePolicyTableHeaders(togglePolicyDetailsModal);
+  if (deviceUser) {
+    // Remove view all hosts link
+    tableHeaders.pop();
+  }
+  const failingResponses: IHostPolicy[] =
+    policies.filter((policy: IHostPolicy) => policy.response === "fail") || [];
+
+  const renderHostPolicies = () => {
+    if (hostPlatform === "ios" || hostPlatform === "ipados") {
+      return (
+        <EmptyTable
+          header={<>Policies are not supported for this host</>}
+          info={
+            <>
+              Interested in detecting device health issues on{" "}
+              {hostPlatform === "ios" ? "iPhones" : "iPads"}?{" "}
+              <CustomLink url={SUPPORT_LINK} text="Let us know" newTab />
+            </>
+          }
+        />
+      );
+    }
+
+    if (policies.length === 0) {
+      return (
```

Comment:
Just double-checking nothing got lost in merge conflicts , should the `EmptyTable` be returned within a `Card` here?